mmeval/metrics/matting_mse.py

```diff
@@ -0,0 +1,90 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import numpy as np
+from typing import Dict, List, Sequence
+
+from mmeval.core import BaseMetric
+
+
+class MattingMSE(BaseMetric):
+    """Mean Squared Error metric for image matting.
+
+    This metric compute per-pixel squared error average across all
+    pixels.
+    i.e. mean((a-b)^2) / norm_const
+
+    .. note::
+
+        Current implementation assume image / alpha / trimap array in numpy
+        format and with pixel value ranging from 0 to 255.
+
+    .. note::
+
+        pred_alpha should be masked by trimap before passing
+        into this metric
+
+    Default prefix: ''
+
+    Args:
+        **kwargs: Keyword parameters passed to :class:`BaseMetric`.
+        norm_const (int): Divide the result to reduce its magnitude.
+            Default to 1000.
+
+    Metrics:
+        - MattingMSE (float): Mean of Squared Error
+
+    Examples:
+
+        >>> from mmeval import MattingMSE
+        >>> import numpy as np
+        >>>
+        >>> mattingmse = MattingMSE()
+        >>> gts = np.random.randint(0, 255, size=(3, 32, 32))
+        >>> preds = np.random.randint(0, 255, size=(3, 32, 32))
+        >>> trimap = np.random.choice(a=(0,128,255), size=(3, 32, 32))
+        >>> mattingmse(preds, gts, trimap)  # doctest: +ELLIPSIS
+        {'MattingMSE': ...}
+    """
+
+    default_prefix = ''
```

Comment:
This can be removed~

Reply:
Thank you for reminding me that it has been deleted in the latest submission.